docs/changehistory/NextVersion.md

```diff
@@ -592,6 +585,12 @@ The signatures to several methods in [BriefcaseManager]($backend) and [Briefcase
 | `BriefcaseManager.downloadBriefcase`     | [RequestNewBriefcaseArg]($backend)                    |                                  |
 | `IModelDb.importSchemas`                 | `LocalFileName[]`                                     | `requestContext` removed         |
 
+## Deprecation of the Settings on iModelApp
+
+The previous `IModelApp.settings` API has been removed in favor of [IModelApp.userPreferences]($frontend). THe updates API will provide a clear distinction between the "settings" within the control of the user (user preferences) and which are within control of the admin of an iModel/iTwin (Workspaces). The separation is intended to make it clear who is capability of modifying and overriding a given setting while making the API easier to use.
```

Comment:
Sorry for nitpicks:
/s/THe updates API/The updated API
/s/simply easy-to-use/simple easy-to-use

Reply:
no problem, fixed.